pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -83,6 +86,7 @@ func init() {
 	DefaultKarmadaControllerManagerImage = fmt.Sprintf("docker.io/karmada/karmada-controller-manager:%s", releaseVer.PatchRelease())
 	DefualtKarmadaWebhookImage = fmt.Sprintf("docker.io/karmada/karmada-webhook:%s", releaseVer.PatchRelease())
 	DefaultKarmadaAggregatedAPIServerImage = fmt.Sprintf("docker.io/karmada/karmada-aggregated-apiserver:%s", releaseVer.PatchRelease())
+	DefaultHostClusterDomain = "cluster.local"
```

Comment:
It could be used by multi command, should we move it to `pkg/karmadactl/options/global.go`?